src/main/java/Database/currentUser.csv

```diff
@@ -0,0 +1 @@
+﻿"currentUserID","isAdmin","email","password","postIDs","favouritesIDs"
```

Comment:
Do we really need to store currentUser in a csv? if we use it so often, can we just have it in the program after a user is logged in?

Reply:
I don't think we do, since CurrentUser will always be set to one specific User or null. If A use case ever needs to pull the currentUser we can just use a getter method. Though currently the getCurrentUser() method in the entity only returns the users email, so that might need to be revised.

Reply:
Yeah makes sense I just added it for now so I could actually use my use case since currentUser has not been implemented!

Reply:
Okay I changed my code to use the static variable CurrentUser.getCurrentUser() instead!